xml/en/docs/syntax.xml

```diff
@@ -39,18 +69,41 @@ minutes, hours, days and so on, using the following suffixes:
 </table>
 </para>
 
+</section>
+
+
+<section id="combine" name="Combining multiple units">
+
 <para>
 Multiple units can be combined in a single value
 by specifying them in the order from the most to the least significant,
 and optionally separated by whitespace.
 For example, “<literal>1h 30m</literal>” specifies the same time
 as “<literal>90m</literal>” or “<literal>5400s</literal>”.
+</para>
+
+<para>
+<list type="bullet">
+
+<listitem>
+<para>
 A value without a suffix means seconds.
```

Comment:
See previous comment.
Either it should be merged with previous section, or this line is wrong for size/offset.